flink-cdc-runtime/src/test/java/org/apache/flink/cdc/runtime/operators/transform/PreTransformOperatorTest.java

```diff
@@ -526,4 +526,39 @@ void testMetadataTransform() throws Exception {
                         new StreamRecord<>(
                                 new CreateTableEvent(METADATA_TABLEID, EXPECTED_METADATA_SCHEMA)));
     }
+
+    @Test
+    void testMultiTransform() throws Exception {
+        PreTransformOperator transform =
+                PreTransformOperator.newBuilder()
+                        .addTransform(
+                                CUSTOMERS_TABLEID.identifier(),
+                                "col1, upper(col1) col12",
+                                "col1 = 'col1'",
+                                "col2",
+                                "col12",
+                                "key1=value1,key2=value2")
+                        .addTransform(
+                                CUSTOMERS_TABLEID.identifier(),
+                                "col2, upper(col2) col12",
+                                "col1 != 'col1'",
+                                "col2",
+                                "col12",
+                                "key1=value1,key2=value2")
+                        .build();
```

Comment:
Considering this bug is sequential specific, swapping rules to verify it would be nice.